config/routes.rb

```diff
@@ -770,6 +770,16 @@
 
     resources :organisation_settings, path: '/organisation-settings', only: :index
 
+    scope path: 'setup' do
+      resources :organisation_permissions_setup, only: %i[index edit update], path: 'organisation-permissions' do
```

Comment:
At the moment, we are only using the `index` route I've set up here.

I've added all the future routes here too so we can be consistent. In the flow, the URLs for each permission would be `/setup/organisation-permissions/org_permission_id/edit`. Then the results of that form are POSTed to `/setup/organisation-permissions/org_permission_id`. At the end, we GET `/setup/organisation-permissions/check`, POST to `/setup/organisation-permissions/commit` and then finally GET the success page at `/setup/organisation-permissions/success`

Reply:
Makes sense to me